Techie-Org/Application-UI
# [Aazam]-[Admin Added-Intl-Translations-Setup]

Ticket: PROJ-XXXX <br>
Linked PRs: (links to corresponding PRs, optional)

- [ ] Browser testing (Chrome, Firefox, Safari, Edge)<br>
- [ ] Accessibility testing (NVDA + Firefox, Voiceover + Safari)<br>
- [x] I have read and agreed to the process outlined in [Creating a Pull Request](File_Path for creatingPullRequest.md)<br> 
- [x] I ran `npm run extract-intl` to update the messages generated
- [ ] If this is a *downstream* I have double checked that my *destination branch* is my *epic branch*

## Changes

Added setup to generate translation file with `npm run extract-intl` script.

## How to test

Add a new message anywhere in the codebase with the file name `messages.js`.
Run `npm run extract-intl` command in CLI.
Observe that the new messages added will be stored in `app/translations/en.json`.
